docs/src/model_setup/forcing_functions.md

```diff
@@ -97,7 +97,7 @@ of GPU-compiliability.
 
 ### Forcing functions that depend on model fields
 
-Forcing functions may depend on model fields evaluated at the `x, y, z` where forcing is applied.
+Forcing functions may depend on model fields (velocity, tracers or auxiliary fields) evaluated at the `x, y, z` where forcing is applied.
```

Comment:
That's a nice addition. Does `ShallowWaterModel` accept auxiliary fields? Could we add that functionality there? Should we? cc @francispoulin

Reply:
Thanks. I've never used `ShallowWaterModels` myself so wanted to have a play with them before I tried, but it looks like it should be relatively straight forward like with the non hydrostatic model.

Reply:
I've just started looking into implementing this and it looks like `auxiliary_fields` aren't implemented at all in `ShallowWaterModels`